src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -12,37 +13,42 @@
           toggled-state (if (= :on flashlight-state) :off :on)]
       (assoc-in db [:wallet :send-transaction :camera-flashlight] toggled-state))))
 
-(defn- fill-request-details [db address name amount]
+(defn- fill-request-details [db {:keys [address name value symbol gas gasPrice] :as m}]
```

Comment:
How come gasPrice vs gas-price?

Reply:
Can we assert that all of these values: address name value symbol gas gasPrice are present? With a pre condition, say. Since we are updating db.

Reply:
Those values come from parsing an ERC681 URI where values and parameters are left as-is thus `gasPrice`. Didn't seam to provide much value to convert back and forth to clojure case.

Reply:
I like https://github.com/qerub/camel-snake-kebab for this to ensure it is always consistent, but I agree it is outside of scope of PR.